### PR TITLE
fix: allow adding screenshots without baseline

### PR DESCRIPTION
**Pull request by bpraznik**

### CODE REVIEW

File: src/local-server/controller.ts [hunk 1]
```diff
@@ -60,6 +60,11 @@ export class Controller {
   }
 
   private async updateScreenshots(test: Test) {
+    // allow for empty baseline
+    if (!test.baselinePath) {
```
Comment (kien-ht) [inline src/local-server/controller.ts:65]: Since we rely on the current path pattern here, could you add a TODO comment as a heads up?

**Review by kien-ht — CHANGES_REQUESTED** [review]
This commit looks good. However, since the `FAIL_ON_MISSING_BASELINE` option has been available for a while now, the report should handle this scenario. Could you update the commit title to reflect this as a fix instead of a feat? I’d be happy to merge after your modifications. Thanks!

File: src/local-server/controller.ts (hunk 1 above)
Comment (bpraznik) [reply src/local-server/controller.ts:65]: Added a comment

**Comment by kien-ht** [comment]
I mean the commit message should be a `fix`. Since this package uses semantic release, which determines version tags based on commit messages. Sorry for the confusion.

**Review by kien-ht — APPROVED** [review]

**Comment by bpraznik** [comment]
Changed the commit message

**Comment by kien-ht** [comment]
Thanks for your contribution

**Pull request merged**